docs/api/templates.md

```diff
@@ -155,29 +155,29 @@ curl -X GET http://coder-server:8080/api/v2/organizations/{organization}/templat
-| » created_by_id                    | string                            | false    |              |                                            |
-| » created_by_name                  | string                            | false    |              |                                            |
-| » default_ttl_ms                   | integer                           | false    |              |                                            |
-| » description                      | string                            | false    |              |                                            |
-| » display_name                     | string                            | false    |              |                                            |
-| » icon                             | string                            | false    |              |                                            |
-| » id                               | string                            | false    |              |                                            |
-| » name                             | string                            | false    |              |                                            |
-| » organization_id                  | string                            | false    |              |                                            |
-| » provisioner                      | string                            | false    |              |                                            |
-| » updated_at                       | string                            | false    |              |                                            |
-| » workspace_owner_count            | integer                           | false    |              |                                            |
+| Name                                 | Type                              | Required | Restrictions | Description                                |
+| ------------------------------------ | --------------------------------- | -------- | ------------ | ------------------------------------------ |
+| `*anonymous*`                        | array                             | false    |              |                                            |
+| `» active_user_count`                | integer                           | false    |              | ActiveUserCount is set to -1 when loading. |
+| `» active_version_id`                | string                            | false    |              |                                            |
+| `» allow_user_cancel_workspace_jobs` | boolean                           | false    |              |                                            |
+| `» build_time_stats`                 | `codersdk.TemplateBuildTimeStats` | false    |              |                                            |
+| `»» **additionalProperties**`        | `codersdk.TransitionStats`        | false    |              |                                            |
```

Comment:
We did rename additionalProperties to `[any property]` in another place/representation. Should we do it here as well?

Reply:
Fixed. I hope it's the last place where it appears...